mflowgen/test/test_module.sh

```diff
@@ -296,8 +324,16 @@ function build_module {
     set -x
     mkdir $dirname; cd $dirname
     mflowgen run --design $garnet/mflowgen/$modname
-    # mflowgen stash link --path /home/ajcars/tapeout_stash/2020-0509-mflowgen-stash-ec95d0
     set +x
+
+    # This is currently considered best practice for us I think?
+    if [ "$modname" == "full_chip" ]; then
+        set -x
+        echo "Fetch pre-built RTL from stash"
+        mflowgen stash link --path /home/ajcars/tile-array-rtl-stash/2020-0724-mflowgen-stash-75007d
+        mflowgen stash pull --hash 2fbc7a
+        set +x
+    fi
```

Comment:
This is ok for now, but we'll get rid of it once we fix the tile id problem that's forcing us to pin to this rtl.

Reply:
However, it should probably also apply to `"$modname" == "tile_array"`

Reply:
> However, it should probably also apply to `"$modname" == "tile_array"`

The (my?) current thinking is that all tests operate as submodules of full_chip. E.g. if you want to do tile_array only, it looks like this: "test_module.sh full_chip tile_array --steps lvs", which instantiates tile_array as a subgraph of full_chip, then builds it. Similarly e.g. "test_module.sh full_chip tile_array Tile_PE --steps synthesis" etc.

Reply:
That's true, but it's not impossible that someone would write a tile array test.